lib/txdb/handlers/triggers/handler.rb

```diff
@@ -0,0 +1,50 @@
+module Txdb
+  module Handlers
+    module Triggers
+      class Handler
+        include ResponseHelpers
+
+        class << self
+          def handle_request(request)
+            new(request).handle
+          end
+        end
+
+        attr_reader :request
+
+        def initialize(request)
+          @request = request
+        end
+
+        private
+
+        def handle_safely
+          yield
+        rescue => e
+          respond_with_error(500, "Internal server error: #{e.message}", e)
+        end
+
+        def databases
+          @databases ||= if database_name = request.params['database']
+            Txdb::Config.databases.select do |database|
+              database.database == database_name
```

Comment:
Lol yeah. Unfortunately "database" is a domain object as well as a conceptual thing you connect to and execute queries against :|